lib/api/accounts.js

```diff
@@ -270,16 +345,27 @@ module.exports = function (app) {
 	this.getTopAccounts = (query, error, success) => {
 		const topAccount = new TopAccount();
 
+		// limit = 0 is not valid anymore
+		if (Number(query.limit) === 0) query.limit = Number(100).toString();
```

Comment:
Why we need to cast those to `Number`?

Reply:
We have to be sure that only numbers are compared here.

Reply:
It's wrong, we call that endpoint from top-accounts controller through less-more service - we should know what type we get here, and it's already a number I think. Also - there is no need to call .toString(), as `param` function can handle numbers. Also less-more service is providing default limit of `50`.

Reply:
query.limit is always String here.
We can alternatively use this:
`!query.limit || query.limit === '0'`

The former limit was equal to 100, is there any reason to keep it on 50. BTW, the current frontend request 51.

toString() will be removed, as the API requires Integer

Reply:
Ok, so just leave it as is for now.